### PR TITLE
bugs/use_of_force

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: datavalid.yml [hunk 1]
```diff
@@ -447,12 +451,14 @@ schemas:
...
           unique identifier of use of force report.
-      - name: use_of_force_type
+      - name: use_of_force_description
+      - name: use_of_force_reason
       - name: use_of_force_level
       - name: use_of_force_effective
       - name: age
       - name: years_of_service
       - name: officer_injured
+      - name: citizen_age
```
Comment (pckhoi) [inline datavalid.yml:461]: Let's make the use of force tables highly normalized. The web team will find the same columns repeated across tables confusing. I think `use_of_force_description` and `use_of_force_reason` only need to stay in the `use_of_force` table. `citizen_age` only need to stay in `use_of_force_citizens` table.

Comment (ayyubibrahimi) [reply datavalid.yml:461]: Sorry, I didn't tag you correctly in this issue from the web team where he asked for the tables to be repeated. Please see #292

**Comment by ayyubibrahimi** [comment]
@pckhoi - wrgl pull error at the process data stage that I can't decode

**Comment by ayyubibrahimi** [comment]
@pckhoi It pulled successfully. Thanks for the clarification. Please review whenever you can

**Comment by pckhoi** [comment]
@ayyubibrahimi when you want to trigger the job again, just click the "Details" button

Screen Shot 2022-05-20 at 07 32 53

Then click "Rerun All Jobs" (or "Rerun Failed Jobs" if the workflow failed)

Screen Shot 2022-05-20 at 07 35 03

**Review by pckhoi — APPROVED** [review]
LGTM, please merge from main to trigger the job again.

**Pull request merged**